test/fail_compilation/diag4479.d

```diff
@@ -1,11 +0,0 @@
-// REQUIRED_ARGS: fail_compilation/imports/fail4479.d
```

Comment:
NOTE: this test is now passing...moved to `test/compilable/diag4479.d`. It now passes because the compiler now fully supports importing via filename so long as it doesn't break import order invariance.